docs/docs/FAQ/Machine-Learning-FAQ.md

```diff
@@ -0,0 +1,125 @@
+[openclip retrieval results](https://github.com/mlfoundations/open_clip/blob/main/docs/openclip_retrieval_results.csv).
+
+</details>
+
+<details>
+  <summary>How to apply?</summary>
+
+1. In the browser, log in to your Admin user
+2. Go to the url [http://localhost:2283/admin/system-settings?open=job-settings](http://localhost:2283/admin/system-settings?open=job-settings)
+3. Go to Machine Learning Settings
+4. change the models and save
+5. rerun the jobs of ML to take effect
+
+</details>
+
+:::note
+Changing a model does not delete the previous model, pay attention to delete the old model to save space in drive.
+Models are stored in IMMICH_MODEL-CACHE.
+:::
+
+### I would like to use the Clip model i created, how can i do that?
+
+you can't, because we have a [whitelist](https://github.com/immich-app/immich/blob/main/machine-learning/app/models/constants.py) of models in the server code that has each model’s [dimension size](https://github.com/immich-app/immich/blob/main/server/src/domain/smart-info/smart-info.constant.ts).  We use this to update the index to the right dim size when the model changes so custom models aren’t supported.
+The [export script](https://github.com/immich-app/immich/tree/main/machine-learning/export) in Immich Github page is meant for us to be able to automate exporting and uploading to Huggingface you can send a [pull request](https://github.com/immich-app/immich/pulls) via Github to add this model to Immich check if Immich already have this model in the list of models in [Huggingface](https://huggingface.co/immich-app) before you send pull request.
+([Sources](https://discord.com/channels/979116623879368755/1071165397228855327/1187088114599071855))
+
+### I don't speak English, is there a way I can use the Smart Serach in my language?
+
+Yes, you can get such an option, to do this you need to check what is the best model according to your language, you can do it by checking the [table](https://github.com/mlfoundations/open_clip/blob/main/docs/openclip_multilingual_retrieval_results.csv).
+
+:::note
+If you found a suitable model for your language and you want to use it you may not be able to because Immich didn't add it to their list of models in [Huggingface](https://huggingface.co/immich-app), you can make it happen by send a [pull request](https://github.com/immich-app/immich/pulls) via Github to add this model to Immich.
+
+It is important to check the level of accuracy of the model and the amount of use of processing power Flops(B) in [this table](https://github.com/mlfoundations/open_clip/blob/main/docs/openclip_results.csv) so you can know it will work well with your system.
+:::
+
+### Does Immich support Facial Recognition in video ?
+
+Although it seems that insightface is able to [recognize faces in video](https://github.com/deepinsight/insightface#arcface-video-demo) as of now the option is not yet implemented in Immich.
```

Comment:
Just say "this is not yet implemented in Immich".

Reply:
Yes, Mertalev also mentioned it in my previous commit, I'm new to everything related to Git and just now started to understand a little how to work with it with help from my brother.
It's Done in the new commitment :)